TINY-10056: Make dialog 'x' close button clickable when dialog is busy (#8892)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

### Fixed
- Fixed an issue that prevented setting the inline dialog `size` to 'medium' or 'normal'. #TINY-10015

> - Fixed an issue that prevented close button to be clicked when dialog is blocked. #TINY-10056

Reviewer: ```suggestion
- Fixed an issue that prevented the close button from being clicked when the dialog was blocked. #TINY-10056
```
